locales/de/pdk.po

```diff
@@ -1,604 +0,0 @@
-# German translations for puppet development kit package.
```

Comment:
This file caused a lot of problems for threads because it was no longer up to date with the current pdk.pot file, and when talking to the i18n folks they were confused why we even had a PO file in our repo as they will be managed by transifex. We should discuss this in more detail if you have concerns, but if we would like both the German PO file and threading the German PO file must be kept up to date and in sync with the pdk.pot file. Keeping a PO file in sync with the POT file without deleting all the existing translations is not a feature provided by the gettext-setup-gem we use because the Transifex workflow creates brand new PO files and populates them with the previous translations that match current strings.